test/acceptance/tests/fixtures/crds/servicedefaults.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: consul.hashicorp.com/v1alpha1
```

Comment:
@ishustava is this an okay place for this fixture? I don't think we need kustomize for these.

Reply:
Yeah, looks good to me!